__tests__/app.test.js

```diff
@@ -91,3 +91,52 @@ describe("GET /api/articles/:article_id", () => {
       });
   });
 });
+
+describe("GET /api/articles", () => {
+  test("200: should return an array of all the articles with comment_counts", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        expect(articles.length).toBe(13);
+        articles.forEach((article) => {
+          expect(article).toHaveProperty("article_id", expect.any(Number));
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).toHaveProperty("created_at", expect.any(String));
+          expect(article).toHaveProperty("votes", expect.any(Number));
+          expect(article).toHaveProperty("article_img_url", expect.any(String));
+          expect(article).toHaveProperty("comment_count", expect.any(String));
+        });
+      });
+  });
+  test("200: should sort articles by created_at in descending order", () => {
+    return request(app)
+      .get("/api/articles")
+      .expect(200)
+      .then(({ body: { articles } }) => {
+        const sortedArticles = articles.sort((a, b) => {
+          return new Date(b.created_at) - new Date(a.created_at);
+        });
+        expect(articles).toEqual(sortedArticles);
+      });
+  });
+
+  test("400: responds with bad request for an invalid request", () => {
```

Comment:
It's great to see you're thinking about these things however /articles/somethinghere is relating to the previous test so this test and the one underneath arn't required as this behaviour has already been tested in your /api/articles/:articleID tests above